### DISCUSSION TITLE
Visual Studio 2022 17.5 版本的新特性碉堡了！

### DISCUSSION THREAD

**Discussion by BYJRK** [BYJRK/DotNet-Discussions#22] [通用]
[视频介绍](https://www.youtube.com/embed/0NiHvdoMBO8)

建议大家了解一下

Preview 版本下载链接：https://visualstudio.microsoft.com/zh-hans/vs/preview/#download-preview

**Comment by Chenyliang** [comment]
省流：
1、git提交信息里关联issue
2、粘性滚动sticky scroll，类似DataGrid的FreezeColumn
3、实时预览md文件
4、vs方式添加文件
5、注释拼写检查
6、docker部署

1234都挺方便的

**Reply by BYJRK** [reply]
我下了个preview版，发现他还少说了一个：

![image](https://user-images.githubusercontent.com/11771667/217989873-4bdd0cc0-9da7-48c0-a868-de3b152922f5.png)

**Comment by Monoideism** [comment]
启动速度感觉慢了好多啊。
第五项，每次忽略之后，关了重进，下面又标拼写错误了，很恼人。
第二项挺好用的。别的什么1，3，4光头视频里介绍了。
